src/filters/turndown.js

```diff
@@ -5,5 +5,41 @@ module.exports = value => {
 		headingStyle: "atx",
 		codeBlockStyle: "fenced"
 	});
+
+	td.keep(["table"]);
+
+	td.addRule("youtube", {
+		filter: ["iframe"],
+		replacement: (innerHTML, node) => {
+			return `{% youtube "${node.getAttribute("src").replace("?feature=oembed", "")}" %}`;
+		}
+	});
+
+	td.addRule("image-and-text", {
+		filter: (node) => {
+			return node.classList.contains("wp-block-media-text");
+		},
+		replacement: (innerHTML, node) => {
+			let contentEl, content, img, image, alt, imagePosition;
+			contentEl = node.querySelector(".wp-block-media-text__content");
+			if (contentEl) {
```

Comment:
Line 25-29 can be simplified to `content = contentEl ? td.turndown(contentEl.innerHTML) : "";`

Reply:
See https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/494#discussion_r612366339 — this filter is being removed as it's only needed for import.